package/MDAnalysis/topology/PDBParser.py

```diff
@@ -303,6 +308,28 @@ def _parseatoms(self):
         masses = guess_masses(atomtypes)
         attrs.append(Masses(masses, guessed=True))
 
+        # Guessed attributes
+        # Need to pull elements from Atom names
+        # Similar to the check for atomtypes function
+        if not any(elements):
+            warnings.warn("Element information absent.")
+            attrs.append(Elements(elements, guessed=True))
```

Comment:
> guessed=True

But you are not guessing these. In addition, I interpreted @IAlibay's suggestion to just not add the `Elements` if they are not all present and valid.